### DISCUSSION TITLE
OpenRadioss history download

### DISCUSSION THREAD

**Discussion by Taylor1987** [OpenRadioss/OpenRadioss#3206] [Q&A]
Hi, 
Does anyone know how to download the older version of openradioss? like version year of 2023, welcome if you can share me a link
we need the older version to compatible with our Linux system

and another question is how to find the released features from version to version updates, as I know rad2rad is not supported is older version, but the developer has claimed maybe someday it can be used, so i just want to catch the new updates 
Thanks

**Comment by OlivierWienholtz** [comment]
Hello,
We don't keep older releases of OpenRadioss.
But you can access to older versions of the code by cloring the OpenRadioss repository & doing a checkout on it:

with 
git log
you can identify the git commitID which interests you
git checkout [git commitID]  permits to obtain the OpenRadioss level you want.

You can then rebuild OpenRadioss with this level.

**Reply by Taylor1987** [reply]
Thanks for you reply, I will try as your guide

**Reply by pcing** [reply]
> Thanks for you reply, I will try as your guide

Hi Taylor,
Did you successfully obtain the 2023 version of radioss through this method？
I use the git log but I can only get commitID of 2025 version
